UPGRADE-4.0.md

```diff
@@ -301,20 +301,20 @@ Form
 
    Before:
 
-   ```php
-   'choice_value' => new PropertyPath('range'),
-   'choice_label' => 'strtoupper',
-   ```
-
-   After:
-
    ```php
    'choice_value' => 'range',
    'choice_label' => function ($choice) {
        return strtoupper($choice);
    },
    ```
 
+   After:
+
+   ```php
+   'choice_value' => new PropertyPath('range'),
+   'choice_label' => 'strtoupper',
```

Comment:
I think this option is inverted as well, i.e. the "before" version for `choice_label` should be the right code since 4.0, isn't it?

Reply:
Good catch.